### ISSUE TITLE
Feature request: Some ideas 

### ISSUE THREAD

**Issue by mburger81** [swimlane/ngx-charts#249]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
In the screenshot the first chart is using ng2-charts and the second is using your ngx-charts, they are sharing the same data about 250 records.

This are some ideas and feature request I have:
a) As you can see there are about 250 circles in the chart, there should be a possibility to disable the circles and the tooltips for the data over the cursor, something like that `tooltip: false`
b) with scale you can decide if the y-axis is always begin from "0" or if is scaled around the min and max value, this is a good idea but there could be a a step more. It would be nice the possibility to set min and max value for y-axis by passing them to the chart. For example if we like to draw a temperature line chart for our weather station we want to have at the y-axis always (-5) from minimum and (+5) from maximum measured value and the also minimum should always be at least 0.

I know this two use cases maybe are not yours, but they should not be hard to implement and in my opinion are very nice and useful.

What do you think?

![image](https://cloud.githubusercontent.com/assets/3778892/23309981/11ab3114-fab1-11e6-89fa-b1c828f30459.png)

![image](https://cloud.githubusercontent.com/assets/3778892/23310462/e6de6b5c-fab2-11e6-897f-d8b27cadf171.png)

**Comment by marjan-georgiev** [comment]
Yes, canvas is natively faster than SVG. SVG gets slow as more elements are added to the page. Canvas does not have that problem.

1. This is a valid request and would improve performance in this case

2. This is also valid, there is another issue requesting this: https://github.com/swimlane/ngx-charts/issues/228

Thanks for looking into this. If you catch anything that could improve performance, please report it.

**Comment by mburger81** [comment]
@marjan-georgiev We are interested in contributing to the project for some simple features, we have experience with canvas but no experience with svg. We know canvas is much more performance but didn't image there would be so a big difference.
The performance difference on our dev destkop system is incredible, and we imagine there is even more difference on a mobile platform, which is a a big use case for use. So honestly we need to understand if there is the possibility to improve performance on the lib? Why did you choose svg over canvas for this  project?

**Comment by marjan-georgiev** [comment]
Performance has been addressed in version 6.0.0.
